### PR TITLE
Lh add post server

### PR DESCRIPTION
**Pull request by lauren-hanson**
Server side of PR that will close ticket 18. 

# POST new article to server 

Steps to test
1. Fetch lh-add-post-server and checkout to it
2. Run the debugger and open SQLite
3. Open postman and POST to http://localhost:8088/posts request with the following new post: {
    "id": 1, 
    "user_id": 2, 
    "category_id": 3, 
    "title": "The standard Lorem Ipsum passage, used since the 1500s", 
    "publication_date": "", 
    "image_url": "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSp-eHz2svNqXTmeZeHKpcxcFauhtIYYMuzjQ7xMa4Aa68if0XZ-EclIrt7MZ034F1oI6k&usqp=CAU", 
    "content": "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.", 
    "approved": 1
}
4. Verify that there is a 201 header and new article prints in the body
5.  Make a GET request to http://localhost:8088/posts to make sure new article has been added to all posts and also verify that new post has been made to Posts table in SQLite.

### CODE REVIEW

File: views/repository.py [hunk 1]
```diff
@@ -345,8 +340,38 @@ def create(resource, new_data):
...
+                new_data['title'], new_data["publication_date"],
+                new_data['image_url'], new_data['content'], new_data['approved'] ))
+
+        id = db_cursor.lastrowid
+
+        new_data['id'] = id
+
+        return json.dumps(new_data)
+
+# def create_post(resource, new_post):
```
Comment (nicholasvaldez) [inline views/repository.py]: please remove the commented out code

**Review by nicholasvaldez — CHANGES_REQUESTED** [review]
Code works and looks great! Just have a question about a possible typo and request to remove commented out code. Nice job!

File: views/repository.py [hunk 2]
```diff
@@ -302,7 +302,7 @@ def get_comments_by_post(value):
             c.content,
             u.first_name,
             u.last_name,
-            u.username
+            u.usernamestas
```
Comment (nicholasvaldez) [inline views/repository.py:305]: is this a typo?

Comment (nicholasvaldez) [inline views/repository.py]: is this a typo?

**Review by nicholasvaldez — CHANGES_REQUESTED** [review]
code works and looks great! Just have a question about a possible typo and request to remove commented out code, but nice job!

File: views/repository.py (hunk 1 above)
Comment (lauren-hanson) [reply views/repository.py]: This has been removed!

File: views/repository.py (hunk 2 above)
Comment (lauren-hanson) [reply views/repository.py]: It sure is! This has been updated.

Comment (nicholasvaldez) [reply views/repository.py]: 👍

**Review by nicholasvaldez — APPROVED** [review]

**Pull request merged**